src/server/index.js

```diff
@@ -9,8 +9,14 @@ const http = require('http');
 const stream = require('stream');
 const fs = require('fs');
 const Promise = require('bluebird');
+// const cron = require('node-cron');
```

Comment:
delete commented code

Reply:
Checking if you mean delete or uncomment?

Reply:
I am assuming that if it is commented out then its not needed?   If its not needed it should be deleted.